tests/test_gpu_openmp.py

```diff
@@ -196,8 +196,8 @@ def test_multiple_loops(self):
         assert op.body.unmaps[6].pragmas[0].value ==\
             ('omp target exit data map(delete: g[0:g_vec->size[0]]'
              '[0:g_vec->size[1]][0:g_vec->size[2]])'
-             ' if(devicerm && (g_vec->size[0] != 0) && (g_vec->size[1] != 0)'
-             ' && (g_vec->size[2] != 0))')
+             ' if(devicerm && g_vec->size[0] != 0 && g_vec->size[1] != 0'
```

Comment:
I liked it better with the parenthesis, looks less reable and bit risky like that